starter-code/index.js

```diff
@@ -1,17 +1,46 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.length++;
```

Comment:
it works if you only push one item every time (in this case it will work). Maybe better to write `this.length = this.items.length` after you pushed x into the array and sorted it